pkg/endpoint/policy.go

```diff
@@ -75,6 +75,14 @@ func (e *Endpoint) allowIngressIdentity(owner Owner, id identityPkg.NumericIdent
 	return e.Consumable.AllowIngressIdentityLocked(policy.GetConsumableCache(), id)
 }
 
+// allowEgressConsumer allows security identity id to be communicated to by
```

Comment:
Update the comment to match the function name

Reply:
Consumer is still holding onto dear life! ;)